components/actionbutton/stories/actionbutton.stories.js

```diff
@@ -56,8 +58,8 @@ export default {
 			control: "boolean",
 		},
 		hasPopup: {
-			name: "Has popup",
-			description: "If the button triggers a popup action, this should be set to reflect the type of element that pops-up.",
+			name: "Has pop-up",
+			description: "If the button triggers a pop-up action, this should be set to reflect the type of element that pops-up.",
```

Comment:
I would argue that `pops-up` shouldn't be hyphenated. Not a blocker though, and not something you introduced either.